pyiron_lammps/potential.py

```
@@ -0,0 +1,380 @@
+            path (bool): choose whether to return the full path to the potential or just the potential name
+
+        Returns:
+            list: of possible potentials for the element or the combination of elements
+
+        """
+        if isinstance(element, list):
+            element = set(element)
+        elif isinstance(element, str):
+            element = {element}
+        elif not isinstance(element, set):
+            raise TypeError("Only, str, list and set supported!")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)